src/index.ts

```diff
@@ -211,7 +211,19 @@ function realTest<Variables, Return = Variables | boolean>(
             log("eventStamp", String(options.eventTimestamp))
             log("endTime", String(timer.endTime))
 
-            return options.eventTimestamp >= timer.endTime
+            if (options.eventTimestamp >= timer.endTime) {
+                const index = options.timers.findIndex(
+                    (timer) => timer.path === path
+                )
+
+                if (index !== -1) {
+                    options.timers.splice(index, 1)
+                }
+
+                return true
+            } else {
+                return false
+            }
```

Comment:
Let's write a comment saying what/why we're doing this. Also, the `return false` can be unwrapped from the `else` as a style improvement.